src/platform/Platform.cpp

```diff
@@ -222,6 +223,41 @@ bool fileExists(const std::string& utf8Path) {
     return fs::exists(path);
 }
 
+std::string getFileSha256(const std::string &utf8Path) {

```

Comment:
Could you please consider using `std::vector<uint8_t> Crypto::sha256(const std::string& in)` in `charts/Crypto.cpp` or moving this function into there in a similar style? The idea is to keep all crypto functions in there to avoid including the mbedtls headers anywhere else. Also, it already does the context management etc.